src/components/layout/MainLayout.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import MainHeader from '@layout/main/MainHeader';
+import MainNavigator from '@layout/main/MainNavigator';
+import { ReactNode } from 'react';
+
+type ProtectedRouteProps = {
+  children: ReactNode;
+};
+
+// MainLayout은 상단 헤더와 하단 네비게이터가 있는 레이아웃 입니다.
+const MainLayout: React.FC<ProtectedRouteProps> = ({ children }) => {
+  return (
+    <>
+      <Layout>
+        <MainHeader></MainHeader>
+        <MainContent>{children}</MainContent>
+        <MainNavigator></MainNavigator>
+      </Layout>
+    </>
+  );
+};
+
+export default MainLayout;
+
+const Layout = styled.div`
+  color: ${(props) => props.theme.colors.font}; // 글자 색
+  background-color: ${(props) => props.theme.colors.background}; // 글자 색
```

Comment:
```suggestion
  background-color: ${(props) => props.theme.colors.background}; // 배경 색
```
이 부분은 오타인거 같아서요! 확인 한번 부탁드릴게요!

Reply:
감사합니다!